pages/common/nping.md

```diff
@@ -0,0 +1,36 @@
+# nping
+
+> Network packet generation tool / ping utility.
+> More information: <https://nmap.org/nping/>.
+
+- Ping a specified host using ICMP if the user is allowed to, otherwise using TCP:
+
+`nping {{example.com}}`
+
+- Ping a specified host using ICMP assuming that the user is allowed to do so
```

Comment:
```suggestion
- Ping a specified host using ICMP (assuming the user is allowed to do so):
```
How about this?

Reply:
By "assume" in this case it is meant that nping should assume and execute this command anyway, even if it is not root and actually not allowed to. If you put it in brackets, doesn't it sound like you are only allowed to execute the command if the user is allowed to? Or does that seem wrong to me?

Reply:
Just now read the man page.

Ref. `--privileged (Assume that the user is fully privileged)`

How about this?

```suggestion
- Ping a specified host using ICMP (assuming that the user is fully privileged):
```